### PR TITLE
LINK-2044 | VAT code mapping for VAT percentage

### PR DESCRIPTION
**Pull request by harriris-vincit**
### Description
Talpa's VAT code is semantically equivalent to the VAT percentage used for order item rows. Due to this, a mapping is added between Talpa VAT codes and the common VAT percentages so that the chosen item row / customer group VAT percentage determines what VAT code is sent to Talpa.

### Closes
[LINK-2044](https://helsinkisolutionoffice.atlassian.net/browse/LINK-2044)

[LINK-2044]: https://helsinkisolutionoffice.atlassian.net/browse/LINK-2044?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by terovirtanen** [comment]
**LINKEDEVENTS-API branch is deployed to platta: https://linkedevents-pr919.api.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**LINKEDEVENTS-API branch is deployed to platta: https://linkedevents-pr919.api.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**LINKEDEVENTS-API branch is deployed to platta: https://linkedevents-pr919.api.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**LINKEDEVENTS-API branch is deployed to platta: https://linkedevents-pr919.api.dev.hel.ninja** :rocket::rocket::rocket:

File: registrations/models.py [hunk 1]
```diff
@@ -68,6 +68,21 @@
...
+    VatPercentage.VAT_10.value: "46",
+    VatPercentage.VAT_0.value: "4U",
+}
+
+
+def get_vat_code_mapping_choices():
+    choices = []
+
+    for vat_percentage, vat_label in VAT_PERCENTAGES:
+        choices.append((VAT_CODE_MAPPING[vat_percentage], vat_label))
```
Comment (tuomas777) [inline registrations/models.py:83]:
```suggestion
    choices = [(VAT_CODE_MAPPING[percentage], label) for percentage, label in VAT_PERCENTAGES]
```
IMO a list comprehension is much cleaner for this kind of stuff. ofc feel free to disagree 🙂

**Review by tuomas777 — APPROVED** [review]
Left one minor aesthetic suggestion, other than that LGTM 👍 🐶

File: registrations/models.py (hunk 1 above)
Comment (harriris-vincit) [reply registrations/models.py]: Oh yes, sometimes one just forgets to use the more handy stuff. Changed as suggested.

**Comment by terovirtanen** [comment]
**LINKEDEVENTS-API branch is deployed to platta: https://linkedevents-pr919.api.dev.hel.ninja** :rocket::rocket::rocket:

**Review by jorilindell — APPROVED** [review]

**Comment by terovirtanen** [comment]
**LINKEDEVENTS-API branch is deployed to platta: https://linkedevents-pr919.api.dev.hel.ninja** :rocket::rocket::rocket:

**Pull request merged**